### ISSUE TITLE
Custom OpenID Connect not working after upgrading to 1.14.0

### ISSUE THREAD

**Issue by demcorp** [zorn-v/nextcloud-social-login#83]
Hi,

We are using the app to connect to Nextcloud using an internal Gitlab. After upgrading to 1.14.0 it has stopped working with the following error: `Error: Call to a member function get() on array`

This is the complete trace:

```
[index] Error: Error: Call to a member function get() on array at <<closure>>

0. /var/www/nextcloud/apps/sociallogin/lib/Controller/LoginController.php line 256
   getUserProfile()
1. /var/www/nextcloud/apps/sociallogin/lib/Controller/LoginController.php line 171
   auth("OCA\\SocialLogi ... t", {callback: "http ... }}, "gitlabOIDC", "OpenID Connect")
2. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 166
   customOidc("gitlabOIDC")
3. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 99
   executeController(OCA\SocialLogin\ ... {}, "customOidc")
4. /var/www/nextcloud/lib/private/AppFramework/App.php line 118
   dispatch(OCA\SocialLogin\ ... {}, "customOidc")
5. /var/www/nextcloud/lib/private/AppFramework/Routing/RouteActionHandler.php line 47
   main("OCA\\SocialLogi ... r", "customOidc", OC\AppFramework\ ... {}, {provider: "gitl ... "})
6. <<closure>>
   __invoke({provider: "gitl ... "})
7. /var/www/nextcloud/lib/private/Route/Router.php line 297
   call_user_func(OC\AppFramework\ ... {}, {provider: "gitl ... "})
8. /var/www/nextcloud/lib/base.php line 987
   match("/apps/sociallog ... C")
9. /var/www/nextcloud/index.php line 42
   handleRequest()

GET /apps/sociallogin/custom_oidc/gitlabOIDC
```

In the meantime, we have manually downgraded to 1.13.0.

Regards.

**Comment by johanehnberg** [comment]
Confirmed, we are seeing the same

**Comment by johanehnberg** [comment]
This commit seems to be the reason
https://github.com/zorn-v/nextcloud-social-login/commit/a2e73d2e43506ba642fed64012549f683c7ac39d

**Comment by johanehnberg** [comment]
To downgrade and prevent the faulty upgrade from reinstalling automatically, update version in appinfo/info.xml to 1.14.0. when 1.13.0 is installed.

While this is a hack, there seem to be no migrations or scripted config updates related to the release since 1.13.0, so this should be safe.

**Comment by dominik-th** [comment]
I submitted a pull request to fix this.

**Comment by demcorp** [comment]
> To downgrade and prevent the faulty upgrade from reinstalling automatically, update version in appinfo/info.xml to 1.14.0. when 1.13.0 is installed.
> 
> While this is a hack, there seem to be no migrations or scripted config updates related to the release since 1.13.0, so this should be safe.

I confirm this solution is safe, we have already applied it.

**Comment by zorn-v** [comment]
FIxed in v1.14.1
